### PR TITLE
docs(vue): all of vue docs

### PR DESCRIPTION
**Pull request by mandarini**
All of vue docs

https://nx-dev-git-fork-mandarini-docs-storybook-vue-docs-nrwl.vercel.app/nx-api/vue

### CODE REVIEW

**Review by meeroslav — APPROVED** [review]

**Review by Coly010 — APPROVED** [review]

**Review by isaacplmann — APPROVED** [review]

File: docs/generated/packages/vue/documents/overview.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+{% /tab %}
+{% /tabs %}
+
+### Generate a new project using Vue
+
+To generate a Vue application, run the following:
+
+```bash
```
Comment (ndcunningham) [inline docs/generated/packages/vue/documents/overview.md:52]:
```suggestion
```shell
```

File: docs/generated/packages/vue/documents/overview.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+nx g @nx/vue:app my-app
+```
+
+To generate a Vue library, run the following:
+
+```bash
```
Comment (ndcunningham) [inline docs/generated/packages/vue/documents/overview.md:58]:
```suggestion
```shell
```

File: docs/generated/packages/vue/generators/component.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        "x-priority": "internal"
+      },
+      "unitTestRunner": {
+        "type": "string",
+        "enum": ["vitest", "jest", "none"],
+        "description": "Test runner to use for unit tests.",
+        "x-prompt": "What unit test runner should be used?"
+      }
+    },
+    "required": ["name"],
```
Comment (ndcunningham) [inline docs/generated/packages/vue/generators/component.json:111]: should `project` be required here?

File: docs/generated/packages/vue/documents/overview.md (hunk 1 above)
Comment (mandarini) [reply docs/generated/packages/vue/documents/overview.md]: I changed this to `shell` as suggested, but I think it has the same effect. It's been used interchangeably throughout the repo.

File: docs/generated/packages/vue/documents/overview.md (hunk 2 above)
Comment (mandarini) [reply docs/generated/packages/vue/documents/overview.md]: I changed this to `shell` as suggested, but I think it has the same effect. It's been used interchangeably throughout the repo.

**Review by ndcunningham — APPROVED** [review]

**Review by jaysoo — APPROVED** [review]

**Pull request merged**